docker-mkdebootstrap/docker-mkimage.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+# Cloud build doesn't currently run in privileged mode so this script runs
```

Comment:
Nit, since these are publicly visible we should probably refer to this with the public name, container builder. That way if someone searches this they'll find the right docs.